media-gfx/openvdb/openvdb-3.2.0.ebuild

```diff
@@ -0,0 +1,165 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $id$
+
+EAPI="6"
+PYTHON_COMPAT=( python{2_7,3_4,3_5} )
+
+inherit eutils python-r1
+
+DESCRIPTION="Libs for the efficient manipulation of volumetric data"
+HOMEPAGE="http://www.openvdb.org"
+
+SRC_URI="https://github.com/dreamworksanimation/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MPL-2.0"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="+openvdb-compression doc pdfdoc python X"
+
+REQUIRED_USE="${PYTHON_REQUIRED_USE}
+	pdfdoc? ( doc )"
+
+COMMON_DEPEND="
+	sys-libs/zlib
+	>=dev-libs/boost-1.62:=[${PYTHON_USEDEP}]
+	media-libs/openexr
+	dev-util/cppunit
+	X? ( media-libs/glfw )
+	dev-libs/jemalloc
+	python? ( dev-python/numpy[${PYTHON_USEDEP}] )
+	openvdb-compression? ( >=dev-libs/c-blosc-1.5.0 )
+	dev-libs/log4cplus"
+
+RDEPEND="python? ( ${PYTHON_DEPS} )
+	 ${COMMON_DEPEND}"
+
+DEPEND="${RDEPEND}
+	dev-cpp/tbb
+	doc? (
+		app-doc/doxygen
+		python? ( dev-python/pdoc[${PYTHON_USEDEP}] )
+	)
+	pdfdoc? (
+		app-doc/doxygen[dot,latex]
+		app-text/ghostscript-gpl
+	)
+	${COMMON_DEPEND}"
+
+S="${WORKDIR}/${P}/${PN}"
+
+PATCHES=( "${FILESDIR}"/${P}-numpy-fix.patch
+	  "${FILESDIR}"/${P}-makefile-fixes.patch
+	  "${FILESDIR}"/${P}-build-fixes.patch )
+
+python_module_compile() {
+	mypythonargs=(
+		PYTHON_VERSION=${EPYTHON/python/}
+		PYTHON_INCL_DIR="$(python_get_includedir)"
+		PYCONFIG_INCL_DIR="$(python_get_includedir)"
+		PYTHON_LIB_DIR="$(python_get_library_path)"
+		PYTHON_LIB="$(python_get_LIBS)"
+		PYTHON_INSTALL_INCL_DIR="${D}$(python_get_includedir)"
+		PYTHON_INSTALL_LIB_DIR="${D}$(python_get_sitedir)"
+		NUMPY_INCL_DIR="$(python_get_sitedir)"/numpy/core/include/numpy
+		BOOST_PYTHON_LIB_DIR="${myprefixlibdir}"
+		BOOST_PYTHON_LIB=-lboost_python-${EPYTHON/python/}
+	)
+
+	einfo "Compiling module for ${EPYTHON}."
+	emake python "${myemakeargs[@]}" "${mypythonargs[@]}" EPYDOC=""
+
+	# This is so the correct version of pdoc is used
+	mypyscriptdir=$(python_get_scriptdir)
+}
+
+src_compile() {
+	# The build system is designed to compile and install all
+	# in one go, so doing each phase separately would be be
+	# impossible and the functions would all overlap anyways.
+	# Do nothing
+	return
```

Comment:
No argument from me as it took ages to make it work in an ebuild. I even submitted patches upstream, but they kinda ignored it. Though, I do have good news. They are starting to get CMake files. It's early and no support for python yet. It should be ready for the 4.0.0 release... I hope. At least I made a nice huge comment block explaining it.